### ISSUE TITLE
Exception: IsarError: MdbxError (22): "Invalid argument"

### ISSUE THREAD

**Issue by rifaldhiaw** [isar/isar#173]
**Steps to Reproduce**
Got an exception when trying to run `isar.open`. It's happened both in samples provided here https://github.com/isar/samples and the latest version 1.0.5 in my project.
```
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: IsarError: MdbxError (22): "Invalid argument"
```

**Code sample**
```
final isar = await Isar.open(
    schemas: [QuoteSchema],
    directory: (await getApplicationDocumentsDirectory()).path,
  );
```

**Version**
 - Platform: Android (Running on Xiamoi redmi 6)
 - Flutter version: 2.8.1
 - Isar version: 1.0.5
 - Isar Generator version: 1.0.5

**Comment by rizzi37** [comment]
```

dependencies:
  flutter:
    sdk: flutter
  isar: ^1.0.5 # <--1
  isar_flutter_libs: ^1.0.5 # <--2
  isar_connect: ^1.0.5 # <--3 if you use inspector
  lazy_load_scrollview: 1.3.0
  auto_size_text: ^3.0.0
  flutter_staggered_grid_view: 0.4.0-nullsafety.3
  path_provider: ^2.0.8 

dev_dependencies:
  isar_generator: ^1.0.5 # <--4
  build_runner: any

```

**Comment by leisim** [comment]
Fixed in `2.0.0-dev.0`
